### ISSUE TITLE
Can the tool be used on multiple form fields

### ISSUE THREAD

**Issue by ottlinger** [coliff/bootstrap-show-password-toggle#12]
Is there a way to use this tool  on multiple password fields per page.
Could the name of the field be a parameter somehow?

Thanks

**Comment by ottlinger** [comment]
Adding a second integration with a different id does not work:

```
+                                    <button aria-label="Show password as plain text. Warning: this will display your password on the screen."
+                                            class="d-none" id="toggle-passwordAgain"
+                                            type="button"></button>
```

**Comment by ottlinger** [comment]
As a workaround I copied the JS and CSS and manually edited the styles and the variables to prevent overwriting ..... maybe there's a more generic solution to allow usage of this great tool in multiple password fields.

Thanks

**Comment by coliff** [comment]
heya. sorry for not replying earlier. You can try the recently released v1.1.0 update which includes #6 which may solve this. It should run now on all inputs with `type="password"` rather than requiring `id="password"`. Hope that helps!
